src/ensembl/package.json

```diff
@@ -39,7 +39,8 @@
   },
   "husky": {
     "hooks": {
-      "pre-commit": "lint-staged"
+      "pre-commit": "if git-branch-is master; then exit 1; else lint-staged; fi",
```

Comment:
We probably also want to echo a warning; something like ("no committing into master") 😄 
I wonder if having dedicated shell files to run in these two hooks will be better than typing the script inline.

Reply:
Yes, we'll probably need to have a file for it if we are going to include more lines of code 😄